build.sbt

```diff
@@ -95,7 +97,7 @@ lazy val microservice: Project = Project(appName, file("."))
   .settings(
```

Comment:
Can youy add .disablePlugins(JUnitXmlReportPlugin) to avoid the unstable build issue.